kyuubi-server/src/test/scala/org/apache/kyuubi/server/RestFrontendServiceSuite.scala

```diff
@@ -61,29 +64,62 @@ class RestFrontendServiceSuite extends KyuubiFunSuite{
   }
 
   test("kyuubi rest frontend service http basic") {
+    withKyuubiRestServer {
+      (_, _) =>
+        eventually(timeout(10.seconds), interval(50.milliseconds)) {
+          val html = Source.fromURL("http://localhost:10099/api/v1/ping").mkString
+          assert(html.toLowerCase(Locale.ROOT).equals("pong"))
+        }
+    }
+  }
+
+  test("kyuubi rest frontend service for sessions resource") {
+    withKyuubiRestServer {
+      (_, _) =>
+        val expectedCount = new SessionOpenedCount()
+        expectedCount.setOpenSessionCount(1)
+        val expectedStr = new ObjectMapper().writeValueAsString(expectedCount)
+
+        eventually(timeout(10.seconds), interval(50.milliseconds)) {
+          val html = Source.fromURL("http://localhost:10099/api/v1/sessions/count").mkString
+          assert(html.toLowerCase(Locale.ROOT).equalsIgnoreCase(expectedStr))
+        }
+    }
+  }
+
+  def withKyuubiRestServer(f: (RestNoopServer, RestFrontendService) => Unit): Unit = {
     val server = new RestNoopServer()
     server.stop()
     val conf = KyuubiConf()
     conf.set(KyuubiConf.FRONTEND_REST_BIND_HOST, "localhost")
```

Comment:
we can also specify the port here

Reply:
Here for testing, we can use the default port?

Reply:
We can return host:port in closure so we don't need hard code them. And the `server` seems not useful, seems `frontendService`  is enough. what do you thank ?

Reply:
reasonable, +1